modal/sandbox.py

```diff
@@ -396,7 +397,16 @@ async def _get_task_id(self):
                 await asyncio.sleep(0.5)
         return self._task_id
 
-    async def exec(self, *cmds: str, pty_info: Optional[api_pb2.PTYInfo] = None):
+    async def exec(
+        self,
+        *cmds: str,
+        # Deprecated: internal use only
+        pty_info: Optional[api_pb2.PTYInfo] = None,
+        stdout: StreamType = StreamType.PIPE,
+        stderr: StreamType = StreamType.PIPE,
+        # Internal option to set terminal size and metadata
+        _pty_info: Optional[api_pb2.PTYInfo] = None,
+    ):
         """Execute a command in the Sandbox and return
```

Comment:
Might want to put an assertion/nice error message if the user provides a non-StreamType as stdout/stderr, like an actual file object 😬

Reply:
good idea